js/e2e/index.ts

```diff
@@ -28,76 +35,231 @@ const TESTS = (api: ApiPromise): Extrinsic[] => {
                 api.createType("AssetAvailability" as any),
                 1000000,
             ],
+            verify: async () => {
+                assert(
+                    ((await api.query.assetIndex.assets(42)) as any).isSome,
```

Comment:
Can we use constant instead of this magic asset id?

Reply:
Sure, I'll update this in next hours